packages/clients/src/aptos/aptos.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2023-2024 Shinami Corp.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Aptos, AptosConfig } from "@aptos-labs/ts-sdk";
+
+const NODE_REST_URL = "https://api.shinami.com/aptos/node/v1";
+const NODE_INDEXER_URL =
+  "http://aptos-mainnet-graphql.aptos-node:8080/v1/graphql";
```

Comment:
be sure to adjust this appropriately

Reply:
I don't think we should hard code in the mainnet indexer. By default, making an Aptos client doesn't set anything and requires explicit setting of the `fullnode` and `indexer` urls. When I 
```
const aptosClient = new Aptos(new AptosConfig({}));
console.log(aptosClient.config);
```

I see:
```
{
  network: 'devnet',
  fullnode: undefined,
  faucet: undefined,
  pepper: undefined,
  prover: undefined,
  indexer: undefined,
  client: { provider: [AsyncFunction: aptosClient] },
  clientConfig: {},
  fullnodeConfig: {},
  indexerConfig: {},
  faucetConfig: {}
}

When I 
```
const shinamiAptosClient = createAptosClient(SHINAMI_NODE_ACCESS_KEY);
console.log(shinamiAptosClient.config);
```
I see:
```
{
  network: 'devnet',
  fullnode: 'https://api.shinami.com/aptos/node/v1',
  faucet: undefined,
  pepper: undefined,
  prover: undefined,
  indexer: 'http://aptos-mainnet-graphql.aptos-node:8080/v1/graphql',
  client: { provider: [AsyncFunction: aptosClient] },
  clientConfig: { API_KEY: 'MY_KEY' },
  fullnodeConfig: {},
  indexerConfig: {},
  faucetConfig: {}
}
```

Reply:
I'm not sure where the `Aptos.config.network` value comes into play. It didn't negatively affect the few REST API requests I made to testnet (despite being `devnet`) but it may have some repercussions I'm unaware of.

Reply:
The network that the access key is configured in the Shinami portal determines the routing of the request to the network. `Aptos.config.network` itself is an irrelevant field now, and that's a feature, not a bug. That's why we have this thin wrapper function, `createAptosClient`, to wrap away the implementation details since we are diverging from the AptosConfig.